plugins/inputs/snmp/snmp.go

```diff
@@ -415,6 +415,9 @@ func (t Table) Build(gs snmpConnection, walk bool) (*RTable, error) {
 			// index, and being added on the same row.
 			if pkt, err := gs.Get([]string{oid}); err != nil {
 				return nil, fmt.Errorf("performing get on field %s: %w", f.Name, err)
+			} else if pkt != nil && pkt.PDUType == gosnmp.Report {
+				_, _, oidText, _, _ := SnmpTranslate(pkt.Variables[0].Name)
```

Comment:
Oh, okay I wrongly assumed that the original OID would still be returned in case of error. (Just like when the OID could not be translated, because it is unknown) Shouldn't that be more useful?

Reply:
Discussed during call that not checking the `err` isn't a usual go pattern, so I will change that.